test/fixtures/apps/form-data.js

```diff
@@ -0,0 +1,5 @@
+fly.http.respondWith(function(request) {
+  const formData = new FormData()
+  formData.append('foo', 'bar')
+  return new Response(formData)
```

Comment:
I don't think an example with a `Response` is super relevant. I think `FormData` will be much more useful in requests.